benches/bench_eq.rs

```diff
@@ -27,5 +44,39 @@ pub fn eq_eval(c: &mut Criterion) {
     });
 }
 
-criterion_group!(bench_eq, eq_keygen, eq_eval);
+pub fn eq_batch_eval(c: &mut Criterion) {
+    let mut keys_a: Vec<u8> = Vec::with_capacity(3105);
+    let mut keys_b: Vec<u8> = Vec::with_capacity(3105);
+    let keys_a_pointer: *mut u8 = keys_a.as_mut_ptr();
+    let keys_b_pointer: *mut u8 = keys_b.as_mut_ptr();
+    let n_values: usize = 5;
```

Comment:
As you pointed out by MP, I think that we need multiple benches with different parameters :) This one is fine, but we should evaluate batch keygen for larger batches: the whole point of this batch generation is to be efficient when we preprocess large number of keys. n_values = 100 or 1000 or even 10_000 would be great, if it is fast enough.